tasks/translation-ollama/main.py

```diff
@@ -0,0 +1,85 @@
+from pathlib import Path
+
+import logging
+import zipfile
+
+import fitz
```

Comment:
Is there a need to add this to requirements.txt?

Reply:
fitz is allias for pymupdf